### DISCUSSION TITLE
Why silicoin are using gpu ?

### DISCUSSION THREAD

**Discussion by AlexeiKaDev** [Chia-Network/chia-blockchain#7480] [Ideas]
Why silicoin are using gpu ?
![image](https://user-images.githubusercontent.com/45790487/125564800-497ed5e9-b9be-4e41-87e2-74f49d2eea6a.png)

**Comment by d1Mm** [comment]
wallet GUI is based on chromium web browser witch uses hardware graphics acceleration
